include/interface/blas1_interface.hpp

```diff
@@ -48,20 +48,89 @@ namespace blas {
  * @param _vy  VectorView
  * @param _incy Increment in Y axis
  */
-template <typename Executor, typename ContainerT0, typename ContainerT1,
-          typename T, typename IndexType, typename IncrementType>
+template <int TileSize = 256, bool Tiled = true, typename Executor,
+          typename ContainerT0, typename ContainerT1, typename T,
+          typename IndexType, typename IncrementType>
 typename Executor::Return_Type _axpy(Executor &ex, IndexType _N, T _alpha,
                                      ContainerT0 &_vx, IncrementType _incx,
                                      ContainerT1 &_vy, IncrementType _incy) {
-  auto vx = make_vector_view(ex, _vx, _incx, _N);
-  auto vy = make_vector_view(ex, _vy, _incy, _N);
+  cl::sycl::event ret;
+
+  // For now, only use the tiled version when:
+  //   - _N `mod` TileSize == 0
+  //   - TileSize `mod` _incx == 0
+  //   - _incx == _incy
+  //   - (sizeof(ElemT) * TileSize ) < _N (see SYCL copy bug)
```

Comment:
SYCL copy bug -> ComputeCpp copy bug, its because our implementation, not because the spec!

Reply:
Oops! Good catch, thanks!